src/Filter/FilterWrapper.style.ts

```diff
@@ -0,0 +1,46 @@
+import { Liquidity, makeStyles, Theme } from '@c2fo/components';
+
+export const useStyles = makeStyles<Theme>(theme => ({
+  addFiltersTitle: {
+    padding: theme.spacing(2, 3),
+  },
+  filterButtons: {
+    padding: theme.spacing(4, 3),
```

Comment:
no need for left/right margin once it is fixed in center.

Reply:
Fixed